projects/bash_networking_security/tlsHandshake.sh

```diff
@@ -1 +1,69 @@
 #!/bin/bash
+# author: hwisdom/amank
+
+PUBLIC_INSTANCE_IP=ec2-16-170-201-244.eu-north-1.compute.amazonaws.com
```

Comment:
your solution is great! but please use the public instance **ip** in order to pass the automated test